prow/cmd/phaino/local.go

```diff
@@ -246,7 +250,87 @@ func convertToLocal(ctx context.Context, log *logrus.Entry, pj prowapi.ProwJob,
 		log.Warn("Ignoring resource requirements")
 	}
 
+	// Setup gcloud credentials
+	if opt.useLocalGcloudCredentials || envExists(container.Env, gcloudCredEnvName) {
```

Comment:
Probably should drop `envExists(container.Env, gcloudCredEnvName)`, as I feel this is a bit over smart, would it possible that user does want to attach a different service account? I would expect user explicitly asking for this

Reply:
Fixed.